doc/devel/documenting_mpl.rst

```diff
@@ -106,15 +99,223 @@ html``.
 On Windows, options needs to be set as environment variables, e.g. ``set O=-W
 -j4 & make html``.
 
-Writing new documentation
-=========================
+.. _writing-rest-pages:
+
+Writing ReST pages
+==================
+
+As noted above, most documentation is either in the docstring of individual
```

Comment:
Might be a personal style thing, but I'd get rid of "As noted above" since this page doesn't need to be read from start to end and people might just read bits and pieces.

Reply:
Fair enough.  Used to writing papers where half the reviewers will get mad at you if you repeat yourself, despite the fact people don't read papers very sequentially either ;-)